packages/sync-repo-settings/test/test.ts

```diff
@@ -64,7 +68,7 @@ function nockUpdateBranchProtection(
   repo: string,
   contexts: string[],
   requireUpToDateBranch: boolean,
-  requireCodeOwners = false
```

Comment:
Curious, why change this?

Reply:
The test 'should update settings for a known repository' needs this property to be true for the test to pass.